### PR TITLE
Vulnerabilities Fix for AzureFunctionOnKubernetesV0

### PR DESCRIPTION
**Pull request by v-bsanthanak**
**Task name**: AzureFunctionOnKubernetesV0

**Description**: 

**Documentation changes required:** (Y/N) 
**Added unit tests:** (Y/N) 

**Attached related issue:** (Y/N)  https://dev.azure.com/mseng/AzureDevOps/_workitems/edit/2029520

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected
- [x] node make.js build --task AzureFunctionOnKubernetesV0
- [x] node make.js test --task AzureFunctionOnKubernetesV0

### CODE REVIEW

**Review by rvairavelu — APPROVED** [review]

File: Tasks/AzureFunctionOnKubernetesV0/make.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
         {
             "items": [
 				"node_modules/azure-pipelines-tasks-docker-common/node_modules/azure-pipelines-task-lib",
-				"node_modules/azure-pipelines-tasks-kubernetes-common/node_modules/azure-pipelines-task-lib"
+				"node_modules/azure-pipelines-tasks-kubernetes-common/node_modules/azure-pipelines-task-lib",
+                "node_modules/azure-pipelines-tool-lib/node_modules/azure-pipelines-task-lib"
```
Comment (ozanaydinMSFT) [inline Tasks/AzureFunctionOnKubernetesV0/make.json:7]: I'm not sure this line is correct

**Pull request merged**